microsoft-edge/devtools-guide-chromium/accessibility/preferred-color-scheme-simulation.md

```diff
@@ -10,35 +10,34 @@ keywords: microsoft edge, web development, f12 tools, devtools
 ---
 # Dark or light color scheme simulation  
 
-Many operating systems have a way to display any application in darker or lighter colors.  Having a web product that has a light theme in a dark-mode operating system is grating and can be an accessibility issue for some users.  
+Many operating systems have a way to display any application in darker or lighter colors.  Having a web product that has a light theme in a dark-mode operating system can be hard to read and can be an accessibility issue for some users.  
 
 For a webpage, you can use the [prefers-color-scheme][MDNPrefersColorScheme] CSS Media Query to detect whether the user prefers to display your product in a darker or lighter color scheme.  Then to test the result, use [Microsoft Edge DevTools][DevtoolsIndex] to simulate a change from dark to light mode, without having to change the setting for your entire operating system.  
```

Comment:
It seems like this paragraph is actually an important required action step that belongs in the following numbered list.  The Accessibility Overview article or child articles shows specific example code to do this, together with this command/setting that's in the numbered list.  Not sure if the present link goes to that article.